script/README.md

```diff
@@ -12,3 +12,19 @@ This line will work share the configuration, as the file will be deployed on Git
 ```
 "$id": "https://HPC-buildtest.github.io/schemas/script/script-v0.0.1.schema.json",
 ```
+
+The fields `$id`, `$schema`, `$title`,  `$type` `required`, `propertyNames`,
+and `properties` are required (and tested for) higher level schema attributes.
+In the table below, we show properties defined under "properties". The
+table below is represented in the schema file.
+
+
+| Name | Description | Type | Required for Schema | Required for User | Default |
+| ---- | ----------- | ---- | ------------------- | ----------------- | -------- |
+| type | the type (must be script) | string | true | true | |
```

Comment:
I thought ``type: script`` would be default but looking at https://github.com/HPC-buildtest/schemas/blob/add/metavar-variables/script/script-v0.0.1.schema.json#L13 we are doing a pattern match. Can't we use ``default: script`` similar to https://github.com/HPC-buildtest/schemas/blob/add/metavar-variables/script/script-v0.0.1.schema.json#L29 defined for ``shell`` key or if there is some other way. 

I think user needs to define ``type: script`` in order to detect the schema type. The ``version:`` key in outer level can detect which version to use.

Reply:
Yes that's why type is set to true for both required for schema and required for user. You can't set it as a default because we can't read some random file without "type" and know it's a script. It could be another config test type (that we haven't developed yet).

Reply:
Makes sense to me now, i think it was slight confusion on my part on how the docs are represented key/value in the table field. It is assumed when user read the ``schema`` docs that ``type: schema`` is inferred while the value is not mentioned explicitly in the table field. This looks good to me.